src/components/dashboard/todo/Todo.js

```diff
@@ -8,27 +8,26 @@ import { makeStyles } from '@material-ui/core/styles';
 const useStyles = makeStyles(theme => ({
   root: {
     flexGrow: 1
-  },
-  control: {
-    padding: theme.spacing(2)
   }
 }));
 
 const Todo = props => {
   const [todoTasks, addTodo] = useState([]);
   const classes = useStyles();
   return (
-    <Grid container className={classes.root} spacing={4}>
-      <Grid item xs={4}>
-        <TodoList />
+    <div className={classes.root}>
+      <Grid container spacing={3} wrap={'nowrap'}>
+        <Grid item lg={4}>
+          <TodoList />
+        </Grid>
+        <Grid item lg={4}>
+          <InProgress />
+        </Grid>
+        <Grid item lg={4}>
```

Comment:
Extract out magic numbers, since TodoList, InProgress and Completed are going to be standardised in size.